backend/src/review/review.service.ts

```diff
@@ -16,9 +17,27 @@ type ReviewQuery = {
 
 @Injectable()
 export class ReviewService {
-  constructor(private readonly databaseService: DatabaseService) {}
+  constructor(
+    private readonly databaseService: DatabaseService,
+    private reviewSimilarityService: ReviewSimilarityService,
+  ) {}
+
+  private async getExistingRevewsForUser(
+    inputReview: Review,
+  ): Promise<Review[]> {
+    try {
+      return await this.databaseService.sql<Review[]>`SELECT REVIEW
+        FROM review
+        WHERE landlord = ${inputReview.landlord}
```

Comment:
Make sure to normalize the input here. Should be `${inputReview.landlord.toLocaleUpperCase()}`